src/App.vue

```diff
@@ -1,200 +1,232 @@
 
-			if (!this.hadNotifications && this.notifications.length) {
-				this._$icon
-					.animate({opacity: 0.6}, 600)
-					.animate({opacity: 1}, 600)
-					.animate({opacity: 0.6}, 600)
-					.animate({opacity: 1}, 600);
+		isRedThemed: function() {
+			if (OCA.Theming && OCA.Theming.color) {
+				var hsl = this.rgbToHsl(OCA.Theming.color.substring(1, 3),
+					OCA.Theming.color.substring(3, 5),
+					OCA.Theming.color.substring(5, 7))
+				var h = hsl[0] * 360
+				return (h >= 330 || h <= 15) && hsl[1] > 0.7 && (hsl[2] > 0.1 || hsl[2] < 0.6)
 			}
+			return false
+		},
+
+		rgbToHsl: function(r, g, b) {
+			r = parseInt(r, 16) / 255; g = parseInt(g, 16) / 255; b = parseInt(b, 16) / 255
+			var max = Math.max(r, g, b); var min = Math.min(r, g, b)
+			var h; var s; var l = (max + min) / 2
+
+			if (max === min) {
+				h = s = 0
+			} else {
+				var d = max - min
+				s = l > 0.5 ? d / (2 - max - min) : d / (max + min)
+				switch (max) {
+				case r: h = (g - b) / d + (g < b ? 6 : 0); break
+				case g: h = (b - r) / d + 2; break
+				case b: h = (r - g) / d + 4; break
+				}
+				h /= 6
+			}
+
+			return [h, s, l]
+		},
+
+		/**
+			 * Performs the AJAX request to retrieve the notifications
+			 */
+		_fetch: function() {
+			axios
+				.get(OC.linkToOCS('apps/notifications/api/v2', 2) + 'notifications')
+				.then(response => {
+					if (response.status === 204) {
+						// 204 No Content - Intercept when no notifiers are there.
+						this._shutDownNotifications()
+					} else if (!_.isUndefined(response.data) && !_.isUndefined(response.data.ocs) && !_.isUndefined(response.data.ocs.data) && _.isArray(response.data.ocs.data)) {
+						this.notifications = response.data.ocs.data
+					} else {
+						console.info('data.ocs.data is undefined or not an array')
+					}
+				})
+				.catch(err => {
+					if (!err.response) {
+						console.info('No response received, retrying')
+						return
+					} else if (err.response.status === 503) {
+						// 503 - Maintenance mode
+						console.info('Shutting down notifications: instance is in maintenance mode.')
+					} else if (err.response.status === 404) {
+						// 404 - App disabled
+						console.info('Shutting down notifications: app is disabled.')
+					} else {
+						console.info('Shutting down notifications: [' + err.response.status + '] ' + err.response.statusText)
+					}
 
-			this.hadNotifications = this.notifications.length > 0;
+					this._shutDownNotifications()
+				})
+		},
+
+		_backgroundFetch: function() {
+			this.backgroundFetching = true
+			this._fetch()
+		},
+
+		/**
+			 * The app was disabled or has no notifiers, so we can stop polling
+			 * And hide the UI as well
+			 */
+		_shutDownNotifications: function() {
+			window.clearInterval(this.interval)
+			this.shutdown = true
 		}
 	}
+}
 </script>
+
+<style scoped>
+	.fade-enter-active,
+	.fade-leave-active,
+	.fade-collapse-enter-active,
+	.fade-collapse-leave-active {
+		transition: opacity 200ms, max-height 200ms;
```

Comment:
```suggestion
		transition: opacity var(--animation-quick), max-height var(--animation-quick);
```

Reply:
Why is this resolved @nickvergessen, shouldn't the change be applied or does it not work? :)

Reply:
Seems i forgot to pull after merging this in the ui